Frontend/src/components/teams/overview/TeamLeaveButton/index.jsx

```diff
@@ -1,19 +1,27 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import Button from 'components/CustomButton';
 import { Modal, ModalTitle } from 'components/Modal';
 import DialogActions from '@mui/material/DialogActions';
 import DialogContent from '@mui/material/DialogContent';
 import DialogContentText from '@mui/material/DialogContentText';
+import Tooltip from '@mui/material/Tooltip';
 
+import { useAuth } from 'shared/context/authContext';
 import TeamService from 'shared/services/Team.service';
 
 function TeamLeaveButton({ team, onLeft }) {
   const [dialogOpen, setDialogOpen] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
+  const [user] = useAuth();
 
   const handleOpenDialog = () => setDialogOpen(true);
   const handleCloseDialog = () => setDialogOpen(false);
 
+  const [onlyAdmin, setOnlyAdmin] = useState();
+  useEffect(() => {
+    setOnlyAdmin(team.adminIds.length === 1 && team.adminIds[0] === user._id);
```

Comment:
What if team has multiple admins?

Reply:
@pal03377 maybe you missed this hah

Reply:
Oh yes, I totally did 😄 I pushed another commit changing this.